spec/system/consumer/shopping/checkout_auth_spec.rb

```diff
@@ -24,6 +24,9 @@
     after { Warden.test_reset! }
 
     before do
+      allow(Flipper).to receive(:enabled?).with(:split_checkout).and_return(true)
+      allow(Flipper).to receive(:enabled?).with(:split_checkout, anything).and_return(true)
```

Comment:
So this enables the split checkout for this spec. But that means that we are not testing the old checkout any more. I reckon that as long as we have the feature toggle, we need to test both.

You could transform these specs into shared examples. I find them a bit cumbersome but I don't know any better / easier way.

Reply:
Great, thanks for that thought. I've implement `shared examples`, but had to consider some exceptions - and used Rspec conditional filters for this - as the expected outcomes are different. My implementation is perhaps a bit cumbersome but it now accounts for both checkouts, till we get rid of our legacy checkout. What do you think?